src/fairseq2/metrics/recorders/_wandb.py

```diff
@@ -11,22 +11,21 @@
 from pathlib import Path
```

Comment:
it should be aligned with fs2 main, it should be rather generic and not critical for mms

Reply:
This looks like a custom use-case for mms to do specific string manipulation.  

I am not sure whether [the new implementation](https://github.com/facebookresearch/fairseq2/blob/wav2vec2_trainer/src/fairseq2/metrics/recorders/wandb.py#L48) made this obsolete due to the introduction of a display name.